tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -1066,6 +1069,7 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_TwoSpeedDXCoolCoil_Only)
     EXPECT_NEAR(state->dataLoopNodes->Node(2).Temp, state->dataLoopNodes->Node(2).TempSetPoint, 0.001);
     // cooling coil air inlet node temp is greater than cooling coil air outlet node temp
     EXPECT_GT(state->dataLoopNodes->Node(3).Temp, state->dataLoopNodes->Node(2).Temp);
+    EXPECT_TRUE(thisSys->m_useNoLoadLowSpeedAirFlow);
```

Comment:
I didn't see any unit tests that actually test the operation, just ensuring this flag is being set properly.  I won't hold this up for that, but in a future effort on here, maybe you could extract out the operation into a worker function that can be easily tested on its own.